### PR TITLE
Fix Test_NsxtL2VpnTunnel for VCD 10.5.1.1+

### PR DESCRIPTION
**Pull request by Didainius**
VCD 10.5.1.23400185 returns 6 asterisks for presharedkey and that causes check to fail in `Test_NsxtL2VpnTunnel`. This PR patches test to match these values.
Note Terraform is not affected as it does not set the PreSharedKey value.

Test `Test_NsxtL2VpnTunnel` passed on both 10.5.1.1 and 10.5.1

### CODE REVIEW

File: .changes/v2.24.0/659-noted.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* Patched `Test_NsxtL2VpnTunnel` to match PresharedKey of VCD 10.5.1.1+ [GH-659]
```
Comment (adambarreiro) [inline .changes/v2.24.0/659-noted.md:1]: Typo in file name

**Review by adambarreiro — APPROVED** [review]
LGTM, thanks for the patch!

File: .changes/v2.24.0/659-noted.md (hunk 1 above)
Comment (Didainius) [reply .changes/v2.24.0/659-noted.md]: Fixed.

**Review by dataclouder — APPROVED** [review]

**Review by lvirbalas — APPROVED** [review]
Thanks!! If you could also add a bit more description to the PR on the context, would be helpful.

**Comment by Didainius** [comment]
> Thanks!! If you could also add a bit more description to the PR on the context, would be helpful.

Improved.

**Pull request merged**